backend/src/routes.js

```diff
@@ -77,4 +78,14 @@ router.post('/ngos', (req, res) => {
     })
 })
 
+// endpoint: get all ngos
+router.get('/ngos', async (req, res) => {
+    try {
+        const ngos = await NGO.find({},'document_state name image description affinities');
```

Comment:
Lets add the document_state = 'Approved' to the find query, and remove it from the projected fields. Sorry for not stating it in the original criteria. At a later date, when we add roles to the app, this will be further configurable.